rust_icu_common/src/lib.rs

```diff
@@ -133,6 +140,18 @@ impl Error {
     }
 }
 
+impl From<ffi::NulError> for Error {
+    fn from(_: ffi::NulError) -> Self {
+        Error::string_with_interior_nul()
```

Comment:
Could we use `anyhow!` and friends here to not lose the error context, by any chance?

Reply:
Not necessary to add it right now, but have a though whether it would be useful.

Reply:
Done. Changed `common::Error::Wrapper` to wrap an `anyhow::Error`.